tensorflow/lite/micro/examples/micro_speech/README.md

```diff
@@ -693,8 +693,7 @@ The following instructions will help you build and deploy the sample to the
     To your installation location. For example: TARGET_TOOLCHAIN_ROOT :=
     /home/myuser/work/CEVA-ToolBox/V18/BX
 4.  Generate the Makefile for the project: /tensorflow$ make -f
-    tensorflow/lite/micro/tools/make/Makefile TARGET=ceva TARGET_ARCH=bx1
```

Comment:
I'd prefer going with TARGET=ceva and TARGET_ARCH=bx1 and having a single kernels/ceva, tools/make/ceva_makefile.inc, tools/make/ext_libs/ceva.inc.

And all the logic to support the different architectures be contained within these directories.